content/blog/2025/01/08/2025-01-08-jenkins-2024-recap.adoc

```diff
@@ -0,0 +1,176 @@
+By providing exposure to open-source practices and expectations, we have enriched the project and open-source community as a whole.
+While participating in these events, contributors have helped update areas of Jenkins documentation and the creation of link:https://stats.jenkins.io/[stats.jenkins.io].
+Even though our Google Summer of Code projects are specific to Jenkins, how to work with an open-source project and engage with the community are invaluable experiences that will expand these contributors' skills beyond Jenkins.
+Both Hacktoberfest and GSoC have become yearly events for Jenkins and we thank them for allowing us to collaborate and to the contributors who help Jenkins continue to be an ever-growing project.
+
+[[outreach]]
+image:/images/post-images/2025/01/08/outreach-and-advocacy.png[Outreach and Advocacy Update]
+
+Throughout 2024, there were several events that Jenkins participated in that allowed us to connect with the Jenkins and extended open-source communities.
+We started the year by gathering in Brussels for FOSDEM and a hugely successful link:https://www.jenkins.io/blog/2024/02/28/jenkins-contributor-summit-and-fosdem-recap/[Jenkins contributor summit].
+The contributor summit provided a space for Jenkins contributors to meet and connect in person, get updates and roadmap plans from Jenkins officers, and work on a handful of different projects together.
+It is not often that we are all able to meet in-person, so these kind of events are hugely invaluable.
+
+In addition to DevOps World and FOSDEM, Jenkins had a presence at the Open Source Summit Europe, where author:gounthar[Bruno Verachten] gave a talk detailing how and why the link:https://www.jenkins.io/blog/2024/10/25/jenkins-tutorial-revamp/[Jenkins tutorial revamp] has been needed and successful.
+He shared details about how incorporating Docker Compose makes the tutorial experience more simplified and beneficial to the user overall, and how it is safer than using the previous Docker-in-Docker method.
+Users benefit from both the comfort of knowing their build is secure and a more straightforward example, allowing them to get started faster.
+
+The last event of the year that involved Jenkins was an link:https://techstrong.tv/videos/cd-pipeline/behind-the-code-jenkins-upgrades-challenges-and-evolution-the-cd-pipeline-ep15[episode of Techstrong TV] that featured author:markewaite[Mark Waite], author:basil[Basil Crow], author:dduportal[Damien Duportal], and author:krisstern[Kris Stern], where they discussed the challenges and successes of the ever-evolving Jenkins project.
+They discussed how upgrades and the constantly changing landscape of development influence the Jenkins project's direction and what it takes to manage a large project like Jenkins.
+Thanks to Techstrong TV for having Jenkins on and to the Jenkins representatives for sharing their insights and anecdotes about the project.
+
+[[modern-ui]]
```

Comment:
It would be good to get a picture into this section reflecting some of the changes in this year

Reply:
Would that help?
![image](https://github.com/user-attachments/assets/c7a53a9c-ab15-48bf-8032-1cbdac9a54d0)

Reply:
ideally something that encompasses the actual changes I think

Reply:
![bento](https://github.com/user-attachments/assets/55c39085-6200-4765-9595-8b42dcdb6bb5)

How is this?

Reply:
Looks great

Reply:
Thanks very much @janfaracik for creating and sharing that, I'll be sure to add it into the post!